docs/release-notes/8.2.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[release-notes-header-8.2.0]]
+== 8.2
+
+[discrete]
+[[release-notes-8.2.0]]
+=== 8.2.0
+
+[discrete]
+[[deprecations-8.2.0]]
+==== Deprecations
+The following endpoints are deprecated ({pull}129448[#129448]) and will be removed in a future release. They will remain active for at least the next 18 months:
+
+* <<bulk-actions-rules-api-create,`/api/detection_engine/rules/_bulk_create`>>
+* <<bulk-actions-rules-api-delete,`/api/detection_engine/rules/_bulk_delete`>>
+* <<bulk-actions-rules-api-update,`/api/detection_engine/rules/_bulk_update`>>
+
+To avoid breakage, we recommend using the <<bulk-actions-rules-api,bulk rule actions>> API instead for similar bulk actions.  You can also use the <<rules-api-create,create>>, <<rules-api-update,update>>, and <<rules-api-delete,delete>> rule APIs to manage rules individually.
+
+[discrete]
+[[breaking-changes-8.2.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+There are no breaking changes in 8.2.0.
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.2.0]]
+==== Features
+* Enables rule previews for indicator match rules ({pull}126651[#126651]).
+* Displays the alerts table when previewing a rule ({pull}127986[#127986]).
+* Introduces a new beta feature, [session-view, Session View]. Session view contextualizes and provides insight into Linux process data ({pull}127828[#127828], {pull}126997[#126997], {pull}127520[#127520], {pull}124575[#124575]).
+* Creates a *Users* page under *Explore* to help you better understand authentication and usage information ({pull}127617[#127617], {pull}127953[#127953], {pull}126434[#126434], {pull}126079[#126079], {pull}128375[#128375], {pull}130030[#130030]).
```

Comment:
```suggestion
* Creates a <<users-page,*Users*>> page under *Explore* to help you better understand authentication and usage information ({pull}127617[#127617], {pull}127953[#127953], {pull}126434[#126434], {pull}126079[#126079], {pull}128375[#128375], {pull}130030[#130030]).
```

Reply:
@jmikell821 I'll commit this edit once you merge https://github.com/elastic/security-docs/pull/1882. Feel free to update the linked text if you'd prefer something different. Thanks!

Reply:
Incorporating this edit